LIMITATIONS.md

```diff
@@ -0,0 +1,16 @@
+## Known Limitations
+
+### Java Client Limitations
+
+- Limited JDK support outside of JDK 11 and above. Unable to build libraries on JDK 21.
+
+### Glide Core Limitations
```

Comment:
Newbie won't know what Glide Core is. Would frame as "All Clients"

Reply:
should be part of the README